drivers/dma/dma_stm32.c

```diff
@@ -379,25 +565,53 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 		LL_DMA_DisableFifoMode(dma, table_ll_stream[id]);
 		LL_DMA_DisableIT_FE(dma, table_ll_stream[id]);
 	}
-#endif
-	return ret;
-}
-
-int dma_stm32_disable_stream(DMA_TypeDef *dma, u32_t id)
-{
-	int count = 0;
-
-	for (;;) {
-		if (!stm32_dma_disable_stream(dma, id)) {
+#endif /* CONFIG_DMA_STM32_V1 */
+#ifdef CONFIG_DMAMUX_STM32
+	if (config->channel_direction != MEMORY_TO_MEMORY) {
+		/* there is also a DMAMUX channel synchro to set */
+		if (mux_channel->sync_en) {
```

Comment:
I don't understand who is setting this.

Reply:
this is the setting of the request line mux configuration register